Userland/Services/Taskbar/TaskbarWindow.cpp

```diff
@@ -84,14 +84,15 @@ TaskbarWindow::TaskbarWindow(NonnullRefPtr<GUI::Menu> start_menu)
     main_widget.set_layout<GUI::HorizontalBoxLayout>();
     main_widget.layout()->set_margins({ 3, 3, 3, 1 });
 
-    auto& start_button = main_widget.add<GUI::Button>("Serenity");
-    start_button.set_font(Gfx::FontDatabase::default_bold_font());
-    start_button.set_icon_spacing(0);
-    start_button.set_fixed_size(80, 22);
+    m_start_button = GUI::Button::construct("Serenity");
+    m_start_button->set_font(Gfx::FontDatabase::default_bold_font());
+    m_start_button->set_icon_spacing(0);
+    m_start_button->set_fixed_size(80, 22);
     auto app_icon = GUI::Icon::default_icon("ladybug");
-    start_button.set_icon(app_icon.bitmap_for_size(16));
-    start_button.set_menu(m_start_menu);
+    m_start_button->set_icon(app_icon.bitmap_for_size(16));
+    m_start_button->set_menu(m_start_menu);
 
+    main_widget.add_child(*m_start_button);
```

Comment:
Is there a particular reason you changed the `main_widget.add` part to adding the child later? Just curious.

Reply:
No, it was just put there out of "style", the creation of the button happens first, then it's added. I can change it if you like :)